src/sort.ts

```diff
@@ -26,11 +30,27 @@ export function sort(document: TextDocument): string {
     const directory = dirname(fileName);
 
     let result;
+    const config = { ...DEFAULT_CONFIGS };
+    const defaultSortStyle = getConfiguration<string>('default-sort-style');
+    for (const languages in config) {
```

Comment:
I know this should be in linting configuration but can you add a blank line before the `for`?

Reply:
:+1: